src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js

```diff
@@ -267,6 +269,33 @@ export default class SpawnPatternModel extends BaseModel {
     }
 
     /**
+     * Destroy the current instance properties
+     *
+     * Useful when changing airports
+     *
+     * @for SpawnPatternModel
+     * @method destroy
+     */
+    destroy() {
```

Comment:
to be honest I've been pretty bad about consistency on those two terms.  The way they _should_ be used is `reset` should be used for reusable classes and `destroy` for simple instances.  But that's not how they're used.  Future project I think.